eng/scripts/New-BuildInfo.ps1

```diff
@@ -261,6 +261,12 @@ function Get-PathsToTest {
             }
         }
 
+        # Always include Azure.Mcp.Server to run ConsolidatedModeTests.cs in all PRs
```

Comment:
As @hallipr called out we don't actually need this.

Reply:
We don't need this yet. I would rather leave it there as a reminder when the time comes.